src/components/Link.js

```diff
@@ -53,11 +53,28 @@ class Link extends React.Component {
       );
     } else {
       return (
-        <router.Link {...pick(this.props, ['route', 'params', 'href', 'scroll', 'passHref'])}>
-          <a className={className} title={title}>
-            {children}
-          </a>
-        </router.Link>
+        <React.Fragment>
+          <style jsx>
+            {`
+              .contributeLink-btn {
```

Comment:
`Link` is a generic component. I don't think it should be responsible to handle the styles for the `contributeLink-btn` that is specific to just one page. What's the motivation to put it here rather that in `CollectiveCover.js`?

Also we must use `styled-components` over `<style/>` whenever possible for new developments (see https://github.com/opencollective/opencollective-frontend/blob/9192aa407afdfc9411cc84d70739eb028f97cc9e/rfc/rfc-002-styled-components.md).